### DISCUSSION TITLE
Is there a way to change the URL variables used for templates?

### DISCUSSION THREAD

**Discussion by tjmills-dev** [supabase/supabase#3273] [Questions] [answered]
I'm using the Supabase GoTrue server, rather than a self-hosted one, but in https://github.com/netlify/gotrue the `MAILER_URLPATHS_INVITE`, `MAILER_URLPATHS_RECOVERY`, etc. variables can be changed. Is there a way to do this on the Supabase-hosted one?

**Comment by J0** [comment, marked as answer]
Hey,

You can do the same in the [environment config file](https://github.com/supabase/supabase/blob/master/docker/.env.example#L49) of the docker setup
